{"name": "Validate login failure with valid username and invalid password -- @1.2 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=129.0.6668.58)\nStacktrace:\n\tGetHandleVerifier [0x00007FF7F2BCFDA5+29557]\n\t(No symbol) [0x00007FF7F2B42240]\n\t(No symbol) [0x00007FF7F29FB6EA]\n\t(No symbol) [0x00007FF7F29CFCD5]\n\t(No symbol) [0x00007FF7F2A7F167]\n\t(No symbol) [0x00007FF7F2A981A1]\n\t(No symbol) [0x00007FF7F2A772A3]\n\t(No symbol) [0x00007FF7F2A412DF]\n\t(No symbol) [0x00007FF7F2A42451]\n\tGetHandleVerifier [0x00007FF7F2EFDCBD+3363469]\n\tGetHandleVerifier [0x00007FF7F2F49B47+3674391]\n\tGetHandleVerifier [0x00007FF7F2F3EAEB+3629243]\n\tGetHandleVerifier [0x00007FF7F2C8FC66+815670]\n\t(No symbol) [0x00007FF7F2B4D6EF]\n\t(No symbol) [0x00007FF7F2B492B4]\n\t(No symbol) [0x00007FF7F2B49450]\n\t(No symbol) [0x00007FF7F2B381FF]\n\tBaseThreadInitThunk [0x00007FFD027C257D+29]\n\tRtlUserThreadStart [0x00007FFD0382AF28+40]\n\n", "trace": "  File \"C:\\Users\\sande\\OneDrive\\Desktop\\Shital\\Trustwell\\NewBDDFramework\\.venv\\Lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"features\\environment.py\", line 19, in after_step\n    allure.attach(context.driver.get_screenshot_as_png(), name='Screenshot', attachment_type=allure.attachment_type.PNG)\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\sande\\OneDrive\\Desktop\\Shital\\Trustwell\\NewBDDFramework\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 839, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\sande\\OneDrive\\Desktop\\Shital\\Trustwell\\NewBDDFramework\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 850, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\sande\\OneDrive\\Desktop\\Shital\\Trustwell\\NewBDDFramework\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\sande\\OneDrive\\Desktop\\Shital\\Trustwell\\NewBDDFramework\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I navigate to homepage", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=129.0.6668.58)\nStacktrace:\n\tGetHandleVerifier [0x00007FF7F2BCFDA5+29557]\n\t(No symbol) [0x00007FF7F2B42240]\n\t(No symbol) [0x00007FF7F29FB6EA]\n\t(No symbol) [0x00007FF7F29CFCD5]\n\t(No symbol) [0x00007FF7F2A7F167]\n\t(No symbol) [0x00007FF7F2A981A1]\n\t(No symbol) [0x00007FF7F2A772A3]\n\t(No symbol) [0x00007FF7F2A412DF]\n\t(No symbol) [0x00007FF7F2A42451]\n\tGetHandleVerifier [0x00007FF7F2EFDCBD+3363469]\n\tGetHandleVerifier [0x00007FF7F2F49B47+3674391]\n\tGetHandleVerifier [0x00007FF7F2F3EAEB+3629243]\n\tGetHandleVerifier [0x00007FF7F2C8FC66+815670]\n\t(No symbol) [0x00007FF7F2B4D6EF]\n\t(No symbol) [0x00007FF7F2B492B4]\n\t(No symbol) [0x00007FF7F2B49450]\n\t(No symbol) [0x00007FF7F2B381FF]\n\tBaseThreadInitThunk [0x00007FFD027C257D+29]\n\tRtlUserThreadStart [0x00007FFD0382AF28+40]\n\n", "trace": "  File \"C:\\Users\\sande\\OneDrive\\Desktop\\Shital\\Trustwell\\NewBDDFramework\\.venv\\Lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"features\\environment.py\", line 19, in after_step\n    allure.attach(context.driver.get_screenshot_as_png(), name='Screenshot', attachment_type=allure.attachment_type.PNG)\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\sande\\OneDrive\\Desktop\\Shital\\Trustwell\\NewBDDFramework\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 839, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\sande\\OneDrive\\Desktop\\Shital\\Trustwell\\NewBDDFramework\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 850, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\sande\\OneDrive\\Desktop\\Shital\\Trustwell\\NewBDDFramework\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\sande\\OneDrive\\Desktop\\Shital\\Trustwell\\NewBDDFramework\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1726872491970, "stop": 1726872491974}, {"name": "When I enter username as \"<userName>\"", "status": "skipped", "start": 1726872494204, "stop": 1726872494204}, {"name": "And I enter password as \"<password>\"", "status": "skipped", "start": 1726872494204, "stop": 1726872494204}, {"name": "And I click on Continue button", "status": "skipped", "start": 1726872494204, "stop": 1726872494204}, {"name": "When I validate the page title", "status": "skipped", "start": 1726872494204, "stop": 1726872494204}, {"name": "When I enter username as \"testUser\"", "status": "skipped", "start": 1726872494204, "stop": 1726872494204}, {"name": "And I enter password as \"\"", "status": "skipped", "start": 1726872494204, "stop": 1726872494204}, {"name": "And I click on Continue button", "status": "skipped", "start": 1726872494204, "stop": 1726872494204}, {"name": "Then I verify invalid login error message is displayed", "status": "skipped", "start": 1726872494204, "stop": 1726872494204}], "parameters": [{"name": "userName2", "value": "testUser"}, {"name": "password2"}], "start": 1726872484292, "stop": 1726872494204, "uuid": "eca56c77-fd02-4a22-a01f-6facaa81d82d", "historyId": "a16c277fe2c7af1caa3643c9be748f78", "fullName": "Calories Calculation functionality: Validate login failure with valid username and invalid password", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "Login"}, {"name": "feature", "value": "Calories Calculation functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}